docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -177,12 +167,11 @@ If document submission is supported, navigate to **Settings > Setup > Tax > _Tax
 
 Prior to testing a tax provider, the merchant or partner test store should have the following configured in the control panel:
 
-* The store default country, found by navigating to **Settings > Setup > Store profile**, is set to a country that is supported by the tax provider
 * The shipping origin address, found by navigating to **Settings > Setup > Shipping**, is configured. This value is included in tax estimate requests
-* The tax provider, found by navigating to **Settings > Setup > Tax**, is enabled
+* The tax provider, found by navigating to **Settings > Setup > Tax > _TaxProviderName_**, is enabled in the relevant countries or subdivisions
 * If document submission is supported, navigate to **Settings > Setup > Tax > _TaxProviderName_** and ensure the **Submit Order Data** checkbox is checked
 
-To test the tax provider connection, navigate to **Settings > Setup > Tax > _TaxProviderName_** in the control panel and click **Test Connection**. This selection will trigger a sample tax estimate request to be sent to the estimate URL provided by the tax provider. If the connection is unsuccessful, merchants can navigate to **Settings > Advanced > Store logs** in the control panel to view the error and its description to assist with triaging the issue.
+To test the tax provider connection, navigate to **Settings > Setup > Tax > _TaxProviderName_** in the control panel and click **Test Connection**. This selection will trigger a sample tax estimate request to be sent to the estimate URL provided by the tax provider. If the connection is unsuccessful, or there is an issue with the response, merchants can navigate to **Settings > Advanced > Store logs** in the control panel to view the error and its description to assist with triaging the issue.
```

Comment:
`Test connection` really validates against a complex tax quote, so we've added some more detail here to explain that the response may also be awry (not just the connection itself).

Reply:
👍